system_metrics_collector/src/system_metrics_collector/linux_memory_measurement_node.hpp

```diff
@@ -67,7 +67,7 @@ class LinuxMemoryMeasurementNode : public system_metrics_collector::PeriodicMeas
     const std::chrono::milliseconds measurement_period,
     const std::string & topic,
     const std::chrono::milliseconds publish_period =
-    PeriodicMeasurementNode::DEFAULT_PUBLISH_WINDOW);
+    PeriodicMeasurementNode::INVALID_PUBLISH_WINDOW);
```

Comment:
Constructor doc needs to be updated with this parameter, can document behavior as well for the default.

Reply:
Done.